### ISSUE TITLE
[Feature Request] more debug output

### ISSUE THREAD

**Issue by NeonLightning** [gameblabla/Ganbare-Natsuki-San#3]
i can't get it to work. some more debug output might be useful. right now when i run it it just goes to next line.

**Comment by gameblabla** [comment]
What are you trying to do exactly ? Porting it to another platform ? You need the assets folder along with the executable.

**Comment by NeonLightning** [comment]
yes i'm trying to get it running on the odroid go advance. is this program sdl1.2 or sdl2?

**Comment by gameblabla** [comment]
It is using SDL 1.2. I was considering making it switch to SDL 2 but it is too much work and the source code isn't doing a good job of separating the different APIs. Maybe i will consider it tho

**Comment by NeonLightning** [comment]
ah that would be my problem then. odd i must have installed the sdl1.2 libs on the last install i had(would explain why some things messed up) as i can't even compile on a clean install because of lack of sdl1.2 on odroid go advance.

**Comment by gameblabla** [comment]
wip branch should have SDL3 version (i could add SDL2 at a later point but it's not a necessity) Closing
